utils/storage/python/Makefile

```diff
@@ -21,7 +21,7 @@
 # SOFTWARE.
 
 test:
-	python3 -m unittest storage_test.MyTest.test_s3
+	TRANSFER_TYPE=S3 bash -c 'python3 -m unittest storage_test.MyTest.test_s3'
```

Comment:
Can you please refactor this for better readability?
test: TRANSFER_TYPE=S3
test:
	python3 -m unittest storage_test.MyTest.test_s3

Reply:
Done in the latest commit.